README.md

```diff
@@ -91,16 +91,16 @@ pip install -r requirements/dev-requirements.txt
 Then:
 
 ```bash
-fab -H commcare-sync-demo.dimagi.com -f deploy/environments/demo.yml deploy
+fab -H commcare-sync-demo2.dimagi.com -f deploy/environments/demo.yml deploy
```

Comment:
one wonders if this DNS should be the long term plan... any reason not to change it back to demo and just reroute the traffic from the old server to the new one?

Reply:
that seems fine to me.